backend/deepchecks_monitoring/ee/bgtasks/object_storage_ingestor.py

```diff
@@ -0,0 +1,172 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+import io
+from urllib.parse import urlparse
+
+import boto3
+import pandas as pd
+import pendulum as pdl
+from pandas.core.dtypes.common import is_integer_dtype
+from sqlalchemy import delete, select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import selectinload
+
+from deepchecks_monitoring.logic.data_ingestion import DataIngestionBackend
+from deepchecks_monitoring.public_models import Organization
+from deepchecks_monitoring.public_models.task import BackgroundWorker, Task
+from deepchecks_monitoring.resources import ResourcesProvider
+from deepchecks_monitoring.schema_models import Model, ModelVersion
+from deepchecks_monitoring.schema_models.column_type import SAMPLE_TS_COL
+from deepchecks_monitoring.schema_models.data_sources import DataSource
+from deepchecks_monitoring.utils import database
+
+__all__ = ['ObjectStorageIngestor']
+
+
+class ObjectStorageIngestor(BackgroundWorker):
+    """Worker to ingest files from s3"""
+
+    def __init__(self, resources_provider: ResourcesProvider):
+        super().__init__()
+        self.ingestion_backend = DataIngestionBackend(resources_provider)
+
+    @classmethod
+    def queue_name(cls) -> str:
+        return 'object_storage_ingestion'
+
+    @classmethod
+    def delay_seconds(cls) -> int:
+        return 0
+
+    async def run(self, task: 'Task', session: AsyncSession, resources_provider: ResourcesProvider):
```

Comment:
@matanper please add log message that will allow us to monitor the ingestion process.
we should have a info level log every time new file was ingested / error log for all exception case